spec/lib/sync_with_dsi_profile_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe SyncWithDsiProfile do
+  let(:controller) { Class.new.extend(SyncWithDsiProfile) }
```

Comment:
This becomes a little cleaner if we call the class directly, as per comment above.